src/org/thoughtcrime/securesms/TransportOption.java

```diff
@@ -11,26 +11,20 @@ public enum Type {
 
   private final int                             drawable;
   private final @NonNull String                 text;
-  private final @NonNull Type                   type;
   private final @NonNull String                 composeHint;
 
   public TransportOption(@NonNull  Type type,
                          @DrawableRes int drawable,
                          @NonNull String text,
                          @NonNull String composeHint)
   {
-    this.type                = type;
     this.drawable            = drawable;
     this.text                = text;
     this.composeHint         = composeHint;
   }
 
   public @NonNull Type getType() {
-    return type;
-  }
-
-  public boolean isType(Type type) {
-    return this.type == type;
+    return Type.NORMAL_MAIL;
```

Comment:
whole TransportOption and TransportOptions classes are probably superfluous, however, there are some callback involved that may do things, so i stopped refactoring here for now to avoid behaviour changes